mods/weather/init.lua

```diff
@@ -79,7 +68,25 @@ end
 local os_time_0 = os.time()
 local t_offset = math.random(0, 300000)
 
-local function update_clouds()
+-- set a default shadow intensity for mgv6 and singlenode
+local function set_default_lighting(players)
+	for _, player in ipairs(players) do
+		local lighting = {
+			shadows = { intensity = 0.33 }
+		}
+		local overrides = weather.on_update(player, { lighting = lighting })
+		if overrides.lighting ~= nil then
+			player:set_lighting(overrides.lighting)
```

Comment:
this will constantly reset the lighting parameters for everyone.
I suppose it's not bad because that's what this mod does anyway (on supported mapgens), but is this intended?
Asking because it didn't do so before.